common-extensions/src/main/java/com/twocoders/util/commonextensions/UuidExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package com.twocoders.util.commonextensions
+
+import java.util.*
+
+val EMPTY_UUID: UUID = UUID.fromString("00000000-0000-0000-0000-000000000000")
```

Comment:
toto vyzera velice specificky. Nie kazdy podla mna nutne povazuje 0000 a 9999 za specialne hodnoty. To je skor z nejakeho projektu odvodene, nie?

Reply:
mno nejde ani tak o nějaký konkrétní projekt, spíš o to, že pokud použiješ tuto "předpřipravenou" konstantu, tak si to tak nadefinuješ pro svůj projekt. Těžko se pro UUID hledají empty a invalid hodnoty, když to oni sami nepřipravily a nechceš to mít nullable :( jako že ani toto nemusí tu za mě není nevyhnutné, jen když si budeš chtít nadefinovat ty sám empty a invalid UUID, tak co tam dáš bez toho aby jsi to obaloval? :)

Reply:
no ide skor o to, ze UUID vacsinou pouzivas spolu s random a tam nemas kontrolu nad tym, co to vygeneruje. S mega mega "stastim" ti random vygeneruje same 0 a si v keli... Je otazne ako sa dostanes (co je to za use case) k invalid alebo empty UUID ale skor by som isiel cestou `nullable` v takom pripade.